checks.d/etcd.py

```diff
@@ -112,4 +112,5 @@ def get_json(self, url, timeout):
                 tags = ["url:{}".format(url)])
             return None
 
+        self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.OK, tags=["url:{}".format(url)])
```

Comment:
Watchout, you'll end up sending two statuses per iteration, one for get_self_metrics and one for get_store_metrics.

Can you ensure to only send one status per iteration per instance ?